CHANGELOG.rst

```diff
@@ -2,6 +2,12 @@
 Changelog
 =========
 
+0.20.0 (YYYY-MM-DD)
+-------------------
+
+* Add :class:`~scrapy_poet.ScrapyPoetRequestFingerprinter`, a request
+  fingerprinter that uses request dependencies in the fingerprint generatoion.
```

Comment:
@Gallaecio seeking suggestions about what we want and don't want to mention here.

Reply:
I think this works, :+1: